raw/src/requests/get_file.rs

```diff
@@ -0,0 +1,32 @@
+use std::borrow::Cow;
+
+use requests::*;
+use types::*;
+
+/// Use this method to get basic info about a file and prepare it for downloading.
+/// For the moment, bots can download files of up to 20MB in size.
+#[derive(Debug, Clone, PartialEq, PartialOrd, Serialize)]
+pub struct GetFile<'s> {
+    file_id: Cow<'s, str>,
+}
+
+impl<'s> Request for GetFile<'s> {
+    type Response = File;
+    type RawResponse = File;
+
+    fn map(raw: Self::RawResponse) -> Self::Response {
+        raw
+    }
+
+    fn name() -> &'static str {
+        "getFile"
+    }
+}
+
+impl<'s> GetFile<'s> {
```

Comment:
I think it should be a method of the [File](https://github.com/telegram-rs/telegram-bot/blob/3ff6e8c210eaa25969bd50b02ec9e5f4fc4a914f/raw/src/types/message.rs#L597)  stucture, right?